### ISSUE TITLE
OBS 28.0.1 Crashes on every other launch

### ISSUE THREAD

**Issue by Willsie-Digital** [obsproject/obs-studio#7317]
### Operating System Info

Windows 11

### OBS Studio Version

28.0.1

### OBS Studio Log URL

https://obsproject.com/logs/T1gH7i2Af4VqGmEr

### OBS Studio Crash Log URL

https://obsproject.com/logs/e7OT1_cRJqkmCTUW

### Expected Behavior

A Blank vanilla OBS install should launch without crashing

### Current Behavior

OBS crashes on every other launch

### Steps to Reproduce

1. launch obs64.exe --portable as administrator
2. complete configuration wizard
3. Exit OBS from controls dock
4. launch obs64.exe --portable as administrator
5. Crash
...

**Comment by WizardCM** [comment]
Your copy of OBS is not blank vanilla, it is being affected by a third party environment variable causing all plugins to load twice. Plugins are not designed to be loaded twice.

**Comment by Willsie-Digital** [comment]
what kind of environment variable would cause this? I just unzipped OBS 28 into a folder created a shortcut and the above happened.

**Comment by Willsie-Digital** [comment]
Can verify it is an environment variable local to my system. this is a non issue.

**Comment by alinsavix** [comment]
I don't suppose y'all want to share the identify of the environment variable that causes this? ;)

**Comment by Willsie-Digital** [comment]
> I don't suppose y'all want to share the identify of the environment variable that causes this? ;)

Sure. I copied OBS 28.0.1 overtop of an OBS 28-rc2 install that had plugins and configs. Removed all files then unzipped a new copy of OBS 28.0.1 into the same folder. Folder name was 'OBS_28_TWILLSIE' whenever OBS launched in that folder name the Tools menu appeared twice and would crash every other launch. If I renamed the folder to 'OBS_28_TWILLS' OBS launched with no issues. 

I don't understand why this would be happening, but I cannot duplicate the issue using any other folder name. So it's likely that this is a variable unique to my environment.
